src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/EditSprintDialog.kt

```diff
@@ -0,0 +1,31 @@
+package com.atlassian.performance.tools.jirasoftwareactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import com.atlassian.performance.tools.jiraactions.api.webdriver.sendKeysWhenClickable
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated
+import java.time.Duration
+
+class EditSprintDialog(
+    private val driver: WebDriver
+) {
+    private val editSprintDialogLocator = By.id("ghx-dialog-edit-sprint")
+    private val sprintNameFieldLocator = By.id("ghx-sprint-name")
+    private val duration = Duration.ofSeconds(30)
```

Comment:
nit: Could be inlined. The "duration" description is already part of it's definition `Duration....`